### PR TITLE
Add instructions for accessing staging instance

### PR DESCRIPTION
**Pull request by JenySadadia**
- Add documentation for requesting a user account on staging instance and related information to use it
- Remove `early-access` doc

### CODE REVIEW

File: doc/staging.md [hunk 1]
```diff
@@ -132,11 +123,10 @@ $ kci user whoami
 
 ## Storage
 
-File storage is managed separately from the API services.  As part of the Early
-Access phase, an AzureFiles token is provided to each user to be able to upload
+File storage is managed separately from the API services. An AzureFiles token is provided to each user to be able to upload
 artifacts.  It is one of the storage solutions supported by KernelCI, you may
 also use SSH or soon any S3-compatible storage.  There is a quota, currently
-5TiB for all the Early Access files.  Old files will be deleted after a while
+5TiB for all the staging files.  Old files will be deleted after a while
```
Comment (JenySadadia) [inline doc/staging.md:129]: Hi @nuclearcat 
I am not sure if this part is true for staging configs.
Could you please provide some insights?

File: doc/staging.md [hunk 2]
```diff
@@ -145,7 +135,7 @@ such as `config/core/<your-username>.yaml`:
 
 ```yaml
 storage:
-  early-access-azure-<your-username>:
+  staging-azure-<your-username>:
     storage_type: azure
     base_url: https://kciapistagingstorage1.file.core.windows.net/
     sas_public_token: "?sv=2022-11-02&ss=f&srt=sco&sp=r&se=2024-10-17T19:19:12Z&st=2023-10-17T11:19:12Z&spr=https&sig=sLmFlvZHXRrZsSGubsDUIvTiv%2BtzgDq6vALfkrtWnv8%3D"
```
Comment (JenySadadia) [inline doc/staging.md:141]: @nuclearcat The same question as above for this.

File: doc/staging.md (hunk 1 above)
Comment (nuclearcat) [reply doc/staging.md:131]: We dont have any policy right now (by request of @padovan ), still have to decide on that after LPC.

Comment (JenySadadia) [reply doc/staging.md:131]: Okay, then should I discard this?

Comment (nuclearcat) [reply doc/staging.md:131]: yes

File: doc/staging.md (hunk 2 above)
Comment (nuclearcat) [reply doc/staging.md:143]: Yes, whole point probably is to rework concept of staging. I dont think we will give separate API accounts anymore (and especially storage).

Comment (JenySadadia) [reply doc/staging.md]: Okay, thanks for confirming.

File: doc/staging.md (hunk 1 above)
Comment (JenySadadia) [reply doc/staging.md]: Okay, thanks. As per our discussion today, I have dropped `storage` section and added details related to accessing staging containers.

**Pull request merged**